scene/animation/animation_player.h

```diff
@@ -207,9 +213,13 @@ class AnimationPlayer : public AnimationMixer {
 	double get_current_animation_position() const;
 	double get_current_animation_length() const;
 
-#ifdef TOOLS_ENABLED
-	void get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const override;
-#endif
+	void set_current_section_with_markers(const StringName &p_start_marker = StringName(), const StringName &p_end_marker = StringName());
+	void set_current_section(double p_start_time = -1, double p_end_time = -1);
+	void reset_current_section();
+
+	double get_current_section_start_time() const;
+	double get_current_section_end_time() const;
+	bool is_playing_section() const;
```

Comment:
Considering that a section always has only one, the prefix current may be unnecessary.
```suggestion
	void set_section_with_markers(const StringName &p_start_marker = StringName(), const StringName &p_end_marker = StringName());
	void set_section(double p_start_time = -1, double p_end_time = -1);
	void reset_section();

	double get_section_start_time() const;
	double get_section_end_time() const;
	bool has_section() const;
```